css/debug.css

```diff
@@ -0,0 +1,364 @@
+*,
+*::before,
+*::after {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
+
+.main-container {
+  width: 100vw;
+  padding: 20px;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  max-height: fit-content;
+}
+
+.container {
+  width: 600px;
+  height: inherit;
+  display: flex;
+  gap: 20px;
+  flex-direction: column;
+}
+
+/***********************/
+/*      POP UP          */
+/***********************/
+.pop-up {
+  width: 300px;
```

Comment:
How about using relative units (rems) for such considerably high amount of pixels ?